src/main/java/ch/njol/skript/variables/FlatFileStorage.java

```diff
@@ -57,7 +57,6 @@
  */
 public class FlatFileStorage extends VariablesStorage {
 	
-	@SuppressWarnings("null")
 	public final static Charset UTF_8 = Charset.forName("UTF-8");
```

Comment:
This is redundant since `StandardCharsets.UTF_8` exists. Replace this & usages and deprecate field for removal.